lib/services/presets.go

```diff
@@ -675,11 +682,16 @@ func AddRoleDefaults(role types.Role) (types.Role, error) {
 		}
 	}
 
+	labels := role.GetMetadata().Labels
+	if role.GetName() == teleport.SystemOktaRequesterRoleName {
+		labels = oldLabels
+	}
```

Comment:
Why do we need this block?

Reply:
The Okta requester is bootstrapped with new labels now to populate the Okta origin field, but if the `okta-requester` role existed beforehand (created by the user), then we want to fail here. This basically makes sure that we're looking at the "user created" labels if they exist.

Reply:
Could you add a comment, please?